Sources/CombineWamp/Model/WampClose.swift

```diff
@@ -0,0 +1,19 @@
+public enum WebSocketCloseCodes: Int {
+    case normal = 1000
+    case error = 3002
+}
+
+public struct WampClose {
+    public let uri: URI
+    public let code: WebSocketCloseCodes
+
+    private init(uri: String, code: WebSocketCloseCodes) {
+        self.uri = .init(unverified: uri)
+        self.code = code
+    }
+
+    public static let systemShutdown = WampClose(uri: "wamp.close.system_shutdown", code: .normal)
```

Comment:
could be enum-cases?

Reply:
WAMP is extensible, one could have custom URI types to describe these actions. An enum would enforce only the standard types, for close and error this is not an issue, but for other data structures we could limit more than WAMP recommends.